utils/io.go

```diff
@@ -36,6 +41,16 @@ func ReadParamsAndOutputFromExt(path string) ([]string, error) {
 		}
 	}
 
+	// explicitly handle line that's too long
+	if scanner.Err() == bufio.ErrTooLong {
+		err_msg := fmt.Sprintf(
+			"ReadParamsAndOutputFromExt() failed on %s with `%s`",
+			path,
+			scanner.Err(),
+		)
+		panic(err_msg)
```

Comment:
That makes sense. What do we think _should_ happen up the stack? This error will mean the 
`extData` (basically the array of Parma estimates) comes back empty. Do we want the summary to fail in that case, or continue to parse and return a summary with no param estimates? In my view, this is kind of a technical issue where it should only happen if we have something configured wrong. And it that case an error (with better formatting and further up the stack perhaps) is the right move. Thoughts?

Reply:
yep - this is something that historically was something I personally did a poor job with (doing things like log.Fatal/panic/etc at the point of failure). For the purposes of the babylon CLI in general I would expect it to bubble all the way to the top then be a fatal error at that point (so exit code non zero + error returned). However, if other tooling was to say wrap this logic elsewhere it could separately process and pass that information to the user.